default/stringtables/en.txt

```diff
@@ -6344,9 +6344,31 @@ METER_HAPPINESS_VALUE_LABEL
+'''The Stability of a colony's population starts out between 1 and 5 after initial colonization depending on how suitable the planet's [[encyclopedia ENVIRONMENT_TITLE]] is for that species. Stability is reduced to 0 after invasion and may also be affected by other factors. A minimum Stability of 5 is necessary for a colony to act as a source of settlers for colonizing other planets. Also [[metertype METER_INDUSTRY]] and [[metertype METER_RESEARCH]] grows slowly if Stability is below 5.
 
-The Target Stability is the stable Stability level a planet's population will approach, given the current factors influencing the planetary Stability. Its value is connected to [[encyclopedia ENC_BUILDING_TYPE]]s built on the planet, [[encyclopedia ENC_TECH]] researched by the empire, [[encyclopedia ENC_SPECIAL]]s linked to the planet, and various other game effects. If these factors change, then the Target Stability value is likely to shift.'''
+The Target Stability is the stable Stability level a planet's population will approach, given the current factors influencing the planetary Stability. Its value is connected to [[encyclopedia ENC_BUILDING_TYPE]]s built on the planet, [[encyclopedia ENC_TECH]] researched by the empire, [[encyclopedia ENC_SPECIAL]]s linked to the planet, and various other game effects. If these factors change, then the Target Stability value is likely to shift.
+
+Common stability modifiers:
+[[STABILITY_MODIFIERS]]
+'''
+
+STABILITY_MODIFIERS
+'''• -/+ 5 Capital Supply Connection - Based on distance in jumps from Capital or Regional Administration Building. Does not affect [[encyclopedia INDEPENDENT_SPECIES_TRAIT_TEXT]] species and empires that have adopted [[policy PLC_CONFEDERATION]] policy.
+• +5 Homeworld Species - species on their own homeworld are more stable
+• +5 Capital Species - Colonies with the same species as the capital are more stable. Does not apply on the Capital itself.
+• +10 Capital - Colony contains [[buildingtype BLD_IMPERIAL_PALACE]].
+• Environment - Bonus or Penalty based on environmental preference of species. Set at gamestart in game rules via the MISC: Stability tab.
+• Planet Size - Bonus or Penalty based on planet size. Set at gamestart in game rules via the MISC: Stability tab.
+• Liked/Disliked Focus
+• Liked/Disliked Policies
+• Liked/Disliked Buildings (On Planet)
+• Liked/Disliked Buildings (In-system)
+• Liked/Disliked Buildings (Off-system)
+• Liked/Disliked Specials (On Planet)
+• Liked/Disliked Specials (In-system)
+• Liked/Disliked Specials (Off-system)
```

Comment:
Liked/Disliked off-system specials do not affect stability, except for liked luxury resources when exported via capital market, which should be covered vi Policies.

For a complete list, World Tree is missing.